recipes/protobuf-c/all/conandata.yml

```
@@ -0,0 +1,9 @@
+sources:
+  "1.4.1":
+    url: https://github.com/protobuf-c/protobuf-c/archive/v1.4.1.tar.gz
+    sha256: 99be336cdb15dfc5827efe34e5ac9aaa962e2485db547dd254d2a122a7d23102
+patches:
+  "1.4.1":
+    - patch_file: "patches/1.4.1-cmake-protobuf-target.patch"
+      patch_description: "Fixup the build"
```

```suggestion
      patch_description: "Fixup the build"
```
It does not help much, please, describe the real meaning, like "Fit Protobuf directory variables to official targets"